omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -91,3 +91,10 @@
 </form>
 
 <div id="patient-search-results"></div>
+
```

Comment:
@wluyima the button is at the bottom of the results table - see discussion here https://issues.openmrs.org/browse/RA-1134